src/components/shoppingList/shoppingList.tsx

```diff
@@ -6,7 +6,7 @@ import styles from './shoppingList.module.css'
 interface ShoppingListProps {
   listId: number
   title: string
-  children?: ReactElement | ReactElement[]
+  children?: ReactElement | ReactElement[] | null
```

Comment:
Needed to fix a bug where no child element was being rendered inside the shopping list. Shopping lists look weird with no child elements.